src/main/scala/org/constellation/domain/snapshotInfo/SnapshotInfoChunk.scala

```diff
@@ -23,4 +23,6 @@ object SnapshotInfoChunk extends Enumeration {
   val SNAPSHOT_CACHE: SnapshotInfoChunk = Val("snapshotCache")
   val LAST_ACCEPTED_TX_REF: SnapshotInfoChunk = Val("lastAcceptedTransactionRef")
   val PUBLIC_REPUTATION: SnapshotInfoChunk = Val("publicReputation")
+  val SNAPSHOT_OWN: SnapshotInfoChunk = Val("snapshotOwn")
```

Comment:
Those two are not properties of `SnapshotInfoChunk` -> single responsibility principle

Reply:
Good point, I'll remove this